tutorials/vue/img.md

```diff
@@ -0,0 +1,139 @@
+To use the `img` tag in vue, you must use the `v-bind:src` directive or `:src` for short.
+Be sure to enclose the image address in single quotes on top of the double quotes.
```

Comment:
Replace with "Remember that `:src` expects a JavaScript expression, so if you want to use a string in `:src` you need to wrap the string in quotes."